atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -0,0 +1,122 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+
+get the image manifest lists from the worker builders. If possible, group them together
+and return them. if not, return the x86_64/amd64 one.
+"""
+
+
+from __future__ import unicode_literals
+from copy import deepcopy
+import requests
+import requests.auth
+import json
+
+from six.moves.urllib.parse import urlparse
+
+from atomic_reactor.plugin import PostBuildPlugin, PluginFailedException
+from atomic_reactor.util import Dockercfg
+
+
+class GroupManifestsPlugin(PostBuildPlugin):
+    key = 'group_manifests'
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow, registries, group=True, goarch=None):
+        """
+        constructor
+
+        :param tasker: DockerTasker instance
+        :param workflow: DockerBuildWorkflow instance
+        :param registries: dict, keys are docker registries, values are dicts containing
+                           per-registry parameters.
+                           Params:
+                            * "secret" optional string - path to the secret, which stores
+                              login and password for remote registry
+        :param group: bool, if true, return the grouped image manifests from the workers
+        :param goarch: dict, keys are platform, values are go language platform names
+        """
+        # call parent constructor
+        super(GroupManifestsPlugin, self).__init__(tasker, workflow)
+        self.group = group
+        self.goarch = goarch or {}
+        self.registries = deepcopy(registries)
+
+    def get_worker_manifest(self, worker_data):
+        worker_digests = worker_data['digests']
+        worker_manifest = []
+
+        for registry, registry_conf in self.registries.items():
+            if registry_conf.get('version') == 'v1':
+                continue
+
+            if not registry.startswith('http://') and not registry.startswith('https://'):
+                registry = 'https://' + registry
+
+            registry_noschema = urlparse(registry).netloc
+            for push_conf_registry in self.workflow.push_conf.docker_registries:
```

Comment:
`delete_from_registry` is actually a noop in orchestrator build because as @vrutkovs pointed out, push_conf is empty in this case. This is being address by https://github.com/projectatomic/atomic-reactor/pull/741.